pmm/pmm2-submodules.groovy

```diff
@@ -137,25 +159,7 @@ pipeline {
                 stash includes: 'CLIENT_URL', name: 'CLIENT_URL'
             }
         }
-        stage('Build client source rpm EL7') {
```

Comment:
@talhabinrizwan Do we support el7 for pmm2 builds? Do we still use el7 as the BUILD_OS, since el9 is the default BUILD_OS used in almost all the builds?

Reply:
Decided to keep the el7 part after confirmation from Talha.